spec/index.html

```diff
@@ -562,15 +547,12 @@ <h2>Blank Node Identifier Issuer State</h2>
     <h2>Canonicalization Algorithm</h2>
 
     <p class="ednote">At the time of writing, there are several open issues that will determine important details of the canonicalization algorithm.</p>
-    <div class="issue" data-number="7"></div>
-    <div class="issue" data-number="8"></div>
     <div class="issue" data-number="10"></div>
     <div class="issue" data-number="11"></div>
-    <div class="issue" data-number="16"></div>
     <div class="issue" data-number="84"></div>
     <div class="issue" data-number="87"></div>
     <div class="issue" data-number="88"></div>
```

Comment:
```suggestion
```
Not hugely important, but as both #87 and #88 pertain to the name of the algorithm, I thought they would be better placed above.